stock_account_operating_unit/model/stock_move.py

```diff
@@ -0,0 +1,131 @@
+# © 2019 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
+# © 2019 Serpent Consulting Services Pvt. Ltd. - Sudhir Arya
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo import exceptions, models
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _generate_valuation_lines_data(
+        self,
+        partner_id,
+        qty,
+        debit_value,
+        credit_value,
+        debit_account_id,
+        credit_account_id,
+        description,
+    ):
```

Comment:
you are missing parameter: svl_id

```suggestion
    def _generate_valuation_lines_data(
        self,
        partner_id,
        qty,
        debit_value,
        credit_value,
        debit_account_id,
        credit_account_id,
        svl_id,
        description,
    ):
```

Reply:
Thank you @kongrattapong, I have committed fix of this issue.